mmv1/products/bigquerydatatransfer/Config.yaml

```diff
@@ -210,6 +211,12 @@ properties:
       **NOTE** : If you are attempting to update a parameter that cannot be updated (due to api limitations) [please force recreation of the resource](https://www.terraform.io/cli/state/taint#forcing-re-creation-of-resources).
     required: true
     custom_flatten: 'templates/terraform/custom_flatten/json_to_string_map.go.tmpl'
+
+  - name: 'sensitiveParamsWoVersion'
```

Comment:
Shouldn't this be a nested field parallel to the WO fied? Is there an explanation for why it isn't and can't be?

Reply:
this is due to how sensitiveParams works

sensitive_params is meant to take in fields for `params`. The api can only take valid params in regards to the `data_source` being used. although it may have been an option to include it in params and remove it before sending the request I felt that it made more sense to leave the values within params to be what's sent and have fields that are not sent outside of the params object.

Reply:
sounds good, i made the change that moves it into sensitive_params while also adding in a check that requires both _wo_version and _wo to be set. https://github.com/GoogleCloudPlatform/magic-modules/pull/13135/commits/3eb3cab85be0b553c41f81a69e7a444c5b6e8ae9

RequiredWith doesn't appear to work.